controllers/controllers_test.go

```diff
@@ -16,16 +16,17 @@ import (
 )
 
 const DEFAULT_ORG_ID string = "4384938490324"
+const DEFAULT_ACCOUNT_NUMBER string = "540155"
 
-func testRequest(method string, path string, orgid string, fakeCaller func(string) SubscriptionsResponse) (*httptest.ResponseRecorder, EntitlementsResponse, string) {
+func testRequest(method string, path string, accnum string, orgid string, fakeCaller func(string) SubscriptionsResponse) (*httptest.ResponseRecorder, EntitlementsResponse, string) {
 	req, err := http.NewRequest(method, path, nil)
 	Expect(err).To(BeNil(), "NewRequest error was not nil")
 
 	ctx := context.Background()
-	ctx = context.WithValue(ctx, identity.Key, identity.XRHID {
-		Identity: identity.Identity {
-			AccountNumber: "540155",
```

Comment:
Why this?
We dont really use account number for anything.

I get that magic nums/stirngs are bad... but this seems to needlessly complicate the signature of testRequest.

Like what does passing it in buy you here?

Reply:
I was trying to make it easier to check whether the account number is "-1" or "", since that's what entitle insights now. Maybe, I can try coming up with a different way of doing this... :thinking:

Reply:
Oh nah.. I probably just missed it in the file. I thought you had externalized it then never used anything but 540155.. if you use -1 somewhere, that makes sense and I am just a dummy :D

Reply:
Yeah, I used -1 as account number in one of the tests :smile: